src/plugins/fast-cjs-plugin.js

```diff
@@ -1,6 +1,6 @@
 const CJS_KEYWORDS = /\b(module\.exports|exports)\b/;
 
-export const ESM_KEYWORDS = /(\bimport\s*(\{|\s['"\w_$])|[\s;]export(\s+(default|const|var|let)[^\w$]|\s*\{))/;
+export const ESM_KEYWORDS = /(\bimport\s*(\{.*?\}\s*from|\s[\w$]\s+from)?\s*['"]|[\s;]export(\s+(default|const|var|let)[^\w$]|\s*\{))/;
```

Comment:
This doesn't match a default export like:

```js
import foo from "bar";
```

I think your suggestion on slack to experiment with alternative solutions is better for the long term. A regex can always lead to false positives :+1:

Reply:
I didn't suggesting something other than a regex :P